src/components/Modal.js

```diff
@@ -67,7 +67,7 @@ export default class Modal extends Component {
         }, [
           showCancel ?
             div({
-              style: _.merge({ marginRight: '1rem' }, Style.elements.button),
+              style: { ...Style.elements.button, marginRight: '1rem' },
```

Comment:
Note: in a few cases like this I deliberately flipped the order. There (currently) aren't any overrides happening here, and I'd argue that this way is what we actually mean.

Reply:
Agreed.